types/stripe-v3/index.d.ts

```diff
@@ -638,8 +638,9 @@ declare namespace stripe {
         save_payment_method?: boolean;
         /**
          * Indicates that you intend to make future payments with this PaymentIntent's payment method.
+         * Valid values are: ["on_session", "off_session"]
          */
-        setup_future_usage?: boolean;
+        setup_future_usage?: string;
```

Comment:
nice catch

it would be beneficial to use union of string literal types instead of enumerating the allowed values in comment. e.g.

```suggestion
        setup_future_usage?: 'off_session' | 'on_session';
```

Reply:
Hey @kaznovac , I just implemented your suggestions and added tests for this change :+1: